README.md

```diff
@@ -144,6 +145,15 @@ The **drum** tool currently supports models with DataRobot-generated Scoring Cod
 or `IRegressionPredictor` interface from [datarobot-prediction](https://mvnrepository.com/artifact/com.datarobot/datarobot-prediction).
 The model artifact must have a **jar** extension.
 
+## Unstructured inference models <a name="unstructured_inference_models"></a>
```

Comment:
Since we're merging into a public repo, I think we should refrain from a documentation for a feature that we didnt even start working on. As long as we keep it undocumented, its unreleased, which it is indeed

Reply:
So I would maybe suggest we document when the feature is at least available on SaaS